README.md

```diff
@@ -1,3 +1,43 @@
+TripleA website: http://triplea-game.github.io/
+
+TripleA Game Features
+=====================
+- Various multiplayer gameplay options:
+  - live online lobby
+  - play by email
+  - play by forum
+- AI for single player play
+- Generic game engine allows you to build your own maps and mods
+
+
+Installing TripleA and Playing
+===============================
+
+Download the installer appropriate for your operating system here: http://github.com/triplea-game/triplea/releases/tag/1.8.0.9, then run the installer, it will guide you through the rest of the game installation.
```

Comment:
Copy/paste error on my part, URL updated in the browser bar and I didn't notice. It was supposed to be .. releases/latest. Now updated in 3435e8